### PR TITLE
fix: Excluded routes are not shown in the server log (baseviews.py)

### PR DESCRIPTION
**Pull request by runoutnow**
The f-string was wrong

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Is CRUD MVC related.
- [ ] Is Auth, RBAC security related.
- [ ] Changes the security db schema.
- [ ] Introduces new feature
- [ ] Removes existing feature

### CODE REVIEW

**Comment by dpgaspar** [comment]
Nice, thks, just a lint problem:
```
flask_appbuilder/baseviews.py:258:91: E501 line too long (93 > 90 characters)
```

**Comment by runoutnow** [comment]
> Nice, thks, just a lint problem:
> 
> ```
> flask_appbuilder/baseviews.py:258:91: E501 line too long (93 > 90 characters)
> ```

ah I thought why use two f-strings for this if you can use one... but that's why =)

**Comment by runoutnow** [comment]
Sorry for the mess in this topic.
I want to make another pull request, but that file is now on this topic...

the changed file for the new pull request doesn't show up when I click "new pull request".

File: flask_appbuilder/security/sqla/manager.py [hunk 1]
```diff
@@ -230,7 +231,11 @@ def update_user(self, user):
             return False
 
     def get_user_by_id(self, pk):
-        return self.get_session.query(self.user_model).get(pk)
+        try:
```
Comment (dpgaspar) [inline flask_appbuilder/security/sqla/manager.py:234]: This seems some sort of retry. What problem are you experiencing?

**Review by dpgaspar — CHANGES_REQUESTED** [review]
LGTM, but please remove the retry on the `get_user_by_id`

**Review by runoutnow — COMMENTED** [review]
> LGTM, but please remove the retry on the `get_user_by_id`

Weeks ago I have removed the retry, but it seems that this pull request is stuck on “change required” while there’s nothing left to change. Perhaps because I previously deleted the file. So I have put a comment in manager.py, merged it and then removed the comment and merged it again. Perhaps github will now determine that the retry on `get_user_by_id` has been removed.

**Pull request closed without merging**